lib/backpex/adapters/ash.ex

```diff
@@ -0,0 +1,112 @@
+defmodule Backpex.Adapters.Ash do
+  @config_schema [
+    resource: [
+      doc: "The `Ash.Resource` that will be used to perform CRUD operations.",
+      type: :atom,
+      required: true
+    ]
+  ]
+
+  @moduledoc """
+  The `Backpex.Adapter` to connect your `Backpex.LiveResource` to an `Ash.Resource`.
+
+  ## `adapter_config`
+
+  #{NimbleOptions.docs(@config_schema)}
+
+  > ### Work in progress {: .error}
+  >
+  > The `Backpex.Adapters.Ash` is currently not usable! It can barely list and show items. We will work on this as we continue to implement  the `Backpex.Adapter` pattern throughout the codebase.
+  """
+
+  use Backpex.Adapter, config_schema: @config_schema
+  require Ash.Query
```

Comment:
If I try to use Backpex in a non Ash project this results in:
> [require Ash.Query](error: module Ash.Query is not loaded and could not be found)

We have to make sure that this module is loaded if Ash is installed only

Reply:
Fixed.